src/health-check/main.cpp

```diff
@@ -132,16 +132,14 @@ class HealthCheckerProcess : public ProtobufProcess<HealthCheckerProcess>
   {
     VLOG(1) << "Check passed";
 
-    // Send a healthy status update on the first success,
-    // and on the first success following failure(s).
-    if (initializing || consecutiveFailures > 0) {
-      TaskHealthStatus taskHealthStatus;
```

Comment:
Oh, got it. Could you follow https://github.com/apache/mesos/blob/master/docs/submitting-a-patch.md to submit the patch at https://reviews.apache.org/groups/mesos/ 
We use reviewboard to review code changes.